extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -126,6 +129,9 @@ public function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 				\Wikia::addAssetsToOutput( 'template_classification_in_edit_js' );
 				\Wikia::addAssetsToOutput( 'template_classification_scss' );
 			}
+		} elseif ( ( new Permissions() )->shouldDisplayBulkActions( $user, $title ) ) {
```

Comment:
Now you're using it in two checks, I'd use a variable for the Permissions object.